app.css

```diff
@@ -1,7 +1,18 @@
 body, html {
     width: 100%;
     height: 100%;
-    background-color:lavender ;
+    background-color:lavender;
+}
+
+body {
+    margin: 0;
```

Comment:
`body` has a default margin, setting its width to 100% meant that the width + the margin for the element would push it beyond the width of the screen. this is what caused most of the uncentering.